charts/longhorn/v1.1.3/Chart.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+name: longhorn
+version: 1.1.3-rc2
+appVersion: v1.1.3-rc2
+kubeVersion: ">=v1.16.0-r0 <v1.22.0-r0"
```

Comment:
@innobead This should be changed to ">= v1.16.0-0, < v1.22.0-0" as per https://github.com/longhorn/longhorn/issues/3356 ?

Reply:
correct, this will be updated.